modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1302,36 +1290,55 @@ RETURN val, val IS NOT :: STRING AS notString
 [[type-predicate-expressions-null]]
 === Type predicate expressions for null
 
-All Cypher types include the `null` value.
-Therefore, `IS ::` will return `true` for all expressions evaluating to `null`, regardless of the type.
+A Cypher type will include the `null` value unless it is appended explicitly with `NOT NULL`.
+Therefore, `IS ::` will return `true` for all expressions evaluating to `null`, which do not explicitly remove it.
 
 .Query
 [source, cypher]
 ----
-RETURN (toFloat(null)) IS :: BOOLEAN AS isBoolean
+WITH (toFloat(null)) AS nullValue
```

Comment:
```suggestion
WITH toFloat(null) AS nullValue
```

I just tested it and the surrounding parenthesis are not needed - unless there is a specific reason to have them? :thinking:

Reply:
Me and Louise added them to the original line to show it was allowed, it was more useful in the other example with an addition I think even if it wasn't needed there either. Might not be needed in all places using the `toFloat` if there's more than one

Reply:
I marked it everywhere I saw it in this PR, and I'd recommend taking it away since there is no extra meaning carried by them. I assume developers know that they can put as many surrounding parenthesis as they wish (and this is not specific to Cypher) ... and if they don't know, we don't need to teach them here :D

Reply:
My initial comment was to add it to the addition to clarify the example (`null + 1 IS NOT :: DATE` is a bit harder to read than `(null + 1) IS NOT :: DATE` even if it means the same thing), it just happened to be added everywhere, I'm fine with removing it from the toFloat examples

Reply:
Agreed! I do think it looks nice on math operations.